includes/class-blocks.php

```diff
@@ -21,12 +21,15 @@ public static function init() {
 		// This is already being called on the init hook, so just add it.
 		self::register_blocks();
 
-		\add_action( 'wp_enqueue_scripts', array( self::class, 'add_data' ) );
-		\add_action( 'enqueue_block_editor_assets', array( self::class, 'add_data' ) );
+		\add_action( 'wp_enqueue_scripts', array( self::class, 'inject_activitypub_options' ) );
+		\add_action( 'admin_print_scripts', array( self::class, 'inject_activitypub_options' ) );
 		\add_action( 'load-post-new.php', array( self::class, 'handle_in_reply_to_get_param' ) );
 		// Add editor plugin.
 		\add_action( 'enqueue_block_editor_assets', array( self::class, 'enqueue_editor_assets' ) );
 		\add_action( 'init', array( self::class, 'register_postmeta' ), 11 );
+		if ( ! \wp_is_block_theme() ) {
+			\add_action( 'the_content', array( self::class, 'maybe_add_fediverse_reactions' ) );
```

Comment:
@obenland I would prefer using one of the comment hooks! The `the_content` is always tricky when using sharing plugins, because it will always be part of the content! It also produces ugly artefacts in combination with Webmentions!

What about `comment_form_before` and `comment_form_comments_closed` instead?


https://github.com/pfefferle/wordpress-webmention/blob/c0312741a98dcf99ad7c21cf3c7ffae89035fb64/includes/class-comment-walker.php#L23

Reply:
(maybe not high priority, because it only adds the empty container `<div>`).

Reply:
yeah, no strong opinion here. @mattwiebe suggested `the_content`, happy to go either direction.